### ISSUE TITLE
Json matcher should work for {"a":1} must /("a")

### ISSUE THREAD

**Issue by etorreborre** [etorreborre/specs2#650]

**Comment by etorreborre** [comment]
Actually it should *not* work because `/("a")` is for matching a value in an array. The fix outputs a better failure message:
```
the object\n{a:1}\ndoesn't contain the value 'a'" +
        "\nThis selector can only be used with an array. Use /(k -> anyValue) if you just want to find the key 'k'
```

**Comment by etorreborre** [comment]
Released in `4.0.4-f9521ff54-20180413093135`.
